src/scripts/create_team.script.ts

```diff
@@ -3,20 +3,20 @@ import { RobotWithClient } from '../hackbot';
 export default (robot: RobotWithClient) => {
 
   robot.respond(/create team (.*)/i, (response) => {
-    const userId = response.message.user.id;
+    const user = robot.adapter.client.rtm.dataStore.getUserById(response.message.user.id);

```

Comment:
Since this is now everywhere could the dataStore be wrapped/exposed at a higher level or something? So we instead have

```typescript
const user = robot.dataStore.getUserById(response.message.user.id);
```

This would also prevent things from breaking as much when they change everything again.